src/Edit.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import './App.css';
+import {Link} from  'react-router-dom';
+
+function EditInterview(event) {
+  
+  const pageUrl = window.location.href
+  const id = pageUrl.substring(35,38);

```

Comment:
Use hook here

Reply:
ok

Reply:
const id = this.props.match.params.id;
on using this, i am getting the id but data get pushed to url also.